f21/ps.html

```diff
@@ -565,10 +565,19 @@ <h2><u> Lecture / Lab 5</u></h2>
 
 
 
-<li > <b>Due Date: 11 October</b> 	<b style="float:right">Reading: Think CS: <a href="https://runestone.academy/runestone/static/thinkcspy/MoreAboutIteration/2DimensionalIterationImageProcessing.html">Section 8.10</a>  &
+<li > <b>Due Date: 14 October</b> 	<b style="float:right">Reading: Think CS: <a href="https://runestone.academy/runestone/static/thinkcspy/MoreAboutIteration/2DimensionalIterationImageProcessing.html">Section 8.10</a>  &
 <a href="https://www.datacamp.com/community/tutorials/python-numpy-tutorial">Datacamp Numpy Tutorial</a>,  <a href="lab3.html">Lab 3</a>,  <a href="lab4.html">Lab 4</a> & <a href="../f20.html">Lecture 4</a></b> <br>
 <p><b>REVIEW from Lecture/Lab 4</b>
-<p>Write a program that creates an image of purple and cyan horizontal stripes. Think color not shade: Purple = red and blue only, no green. Cyan = green and blue only, no red.
+<p>Write a program that creates an image of khaki and rosy horizontal and vertical stripes. 
+  <br>
+  <br>The color codes are below. 
+  <br>
+  rgb(188,143,143), #BC8F8F - Rosy Brown 
+  <br>
+  rgb(240, 230,140) #f0e68c - khaki 
+  <br>
+  Use the color converter to get the RGB% from hex:https://andylangton.co.uk/tools/colour-converter
+  <br>
   <br>Your program should ask the user for the size of your image, the name of the output file, and create a <tt>.png</tt> file of stripes.  For example, if the user enters 10, your program should create a 10x10 image, alternating between purple and cyan stripes.
 
 <p>
```

Comment:
The sample output here only asks for the output file, but the description above says the program should as for both image size and output file, can you please make it consistent?